udplog/_kafka.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Ralph Meijer.
+# See LICENSE for details.
+
+"""
+Kafka support.
+
+This provides a Twisted based publisher to a topic in a Kafka cluster.
+"""
+from __future__ import division, absolute_import
+
+import socket
+
+from kafka import KafkaClient, SimpleProducer
```

Comment:
Is this kafka client async? It can work with twisted and not block things? A quick look it seems like it is not and also not thread safe.